tests/topotests/pim_basic/rp/frr.conf

```diff
@@ -0,0 +1,29 @@
+hostname rp
+!
+frr version 8.4
+frr defaults traditional
+!
+service integrated-vtysh-config
+!
+interface rp-eth0
+ ip address 10.0.30.3/24
+ ip pim
+!
+interface lo
+ ip address 10.254.0.3/32
+ ip pim
+!
+router bgp 65003
+ no bgp ebgp-requires-policy
+ neighbor 10.0.30.1 remote-as external
+ neighbor 10.0.30.1 timers 3 10
+ redistribute connected
+!
+ip pim join-prune-interval 5
+ip pim rp 10.254.0.3
+ip pim register-accept-list ACCEPT
```

Comment:
since you are rewriting the config, might be good to move these under `router pim`

Reply:
sure